h2. Noncompliant Code Example

{code}
if (file != null) {
  if (file.isFile() || file.isDirectory()) {
    ...
  }
}
{code}

{code:title=PHP}
if (condition1) {
  if (condition2) {
    ...
  }
}

{code}
{code:title=ABAP}
IF something.
  IF somethingElse.
    WRITE / 'hello'.
  ENDIF.
ENDIF.
{code}

{code:title=PL/SQL}
IF something THEN
  IF something_else THEN
    -- ...
  END IF;
END IF;
{code}

{code:title=VB6}
If a And b Then
  If c And d Then
    doSomething()
  End If
End If
{code}